app/controllers/candidates/registrations/personal_informations_controller.rb

```diff
@@ -49,12 +48,12 @@ def update
 
       def personal_information_params
         if candidate_signed_in?
-          params.require(:candidates_registrations_personal_information).permit \
+          params.fetch(:candidates_registrations_personal_information, {}).permit \
```

Comment:
What's the reason for replacing require here? Shouldn't the top level param key always be present?

Reply:
If the user is already signed in, then all the fields will be read only and disabled - hence the need for `fetch` - part of me wonders if there should be some logic to skip over that page, but the flip side is the users is still submitting the data, so its good to call out what is getting submitted.